rct229/rulesets/ashrae9012019/ruleset_functions/get_primary_secondary_loops_dict.py

```diff
@@ -64,25 +64,20 @@ def get_primary_secondary_loops_dict(rmi_b):
     ]
     # Initialize variables
     primary_loops = []
-    tmp_primary_secondary_loops_dict = dict()
+    tmp_primary_secondary_loops_dict = {}
 
     # Iterate through cooling type fluid loops
     for chilled_fluid_loop in find_all(
         f'fluid_loops[*][?(@.type="{FLUID_LOOP.COOLING}")]', rmi_b
     ):
         cfl_id = chilled_fluid_loop["id"]
-        if cfl_id in chiller_loop_ids and cfl_id in non_process_chw_coil_loop_ids:
-            # No loop in baseline shall be
-            tmp_primary_secondary_loops_dict = dict()
```

Comment:
After read the RDS again, I think the implementation is missing/misinterpretating one logic.
The RDS says to `break logic and return an empty dictionary (this would be if the system was not modeled as primary-seconday, a fluid_loop will never be connected to a cooling_system.chilled_water_loop)` 

So `if cfl_id in chiller_loop_ids and cfl_id in non_process_chw_coil_loop_ids`, we shall set the `tmp_primary_secondary_loops_dict` to an empty dictionary, and also clear the `primary_loops`, then escape the loop.

We will need to update the current logic and add a test case for that.

Reply:
Addressed.